lib/logger/lib/logger/config.ex

```diff
@@ -93,7 +93,7 @@ defmodule Logger.Config do
   def handle_event({_type, gl, _msg} = event, state) when node(gl) != node() do
     # Cross node messages are always async which also
     # means this handler won't crash in case there is
-    # no logger installed in the other node.
+    # no Logger installed in the other node.
```

Comment:
I guess we need to rephrase this:
```
Cross node messages are always async which also
means this handler won't crash in case Logger
is not installed in the other node.
```